Ghidra/Framework/Generic/src/main/java/ghidra/util/NumericUtilities.java

```diff
@@ -520,20 +520,20 @@ public static String formatNumber(long number, int radix) {
 	 * <table><caption></caption>
 	 * <tr><th>Number</th><th>Radix</th><th>DEFAULT Mode Alias</th><th><i>UNSIGNED</i> Mode Value</th><th><i>SIGNED</i> Mode Value</th></tr>
 	 * <tr><td>&nbsp;</td><td></td><td><i></i></td><td></td><td></td></tr>
-	 * <tr align=right><td>100</td><td>2</td><td><i>UNSIGNED</i></td><td>1100100b</td><td>1100100b</td></tr>
```

Comment:
This table is not rendering correctly for me in my Eclipse hover feature.

Reply:
It appears there was a mistake here as it doesn't line up in the browser either. I'm looking into it now.

Reply:
I've changed a few more things about the table such as aligning the above header and setting all the rows to monospace font so the numbers don't appear funny.